### PR TITLE
Remove tuple output

### PR DESCRIPTION
**Pull request by jfalcou**
Most numeric functions that returned a tuple of wides now return a wide of tuples.
Tuple of wide functions is basically somethign we want for more higher level structural changes like shuffles or some such.
This make the relation between scalar/simd version of any given elementwise function more regular.

### CODE REVIEW

File: include/eve/module/core/regular/ifrexp.hpp [hunk 1]
```diff
@@ -30,7 +41,7 @@ namespace eve
 //!   namespace eve
 //!   {
 //!      template< eve::floating_value T >
-//!      kumi::tuple<T, eve::as_integer<T>> ifrexp(T x) noexcept;
+//!      kumi::tuple<T, eve::as_integer_t<T>> ifrexp(T x) noexcept;
```
Comment (DenisYaroshevskiy) [inline include/eve/module/core/regular/ifrexp.hpp:44]: this seems wrong

File: include/eve/module/core/regular/impl/ifrexp.hpp [hunk 2]
```diff
@@ -33,39 +34,97 @@
 
 namespace eve::detail
 {
-template<floating_ordered_value T>
-EVE_FORCEINLINE constexpr auto
-ifrexp_(EVE_SUPPORTS(cpu_), raw_type const&, T a0) noexcept
-
+template<typename T>
```
Comment (DenisYaroshevskiy) [inline include/eve/module/core/regular/impl/ifrexp.hpp:37]: you rewrote this file?

File: include/eve/module/core/regular/modf.hpp [hunk 3]
```diff
@@ -7,10 +7,29 @@
...
+#include <eve/module/core/regular/zip.hpp>
+#include <eve/arch/platform.hpp>
 
 namespace eve
 {
+  template<typename Options>
+  struct modf_t : elementwise_callable<modf_t, Options, pedantic_option>
+  {
+    template<eve::value T>
+    EVE_FORCEINLINE constexpr eve::result_t<zip,T,T> operator()(T a) const noexcept
```
Comment (DenisYaroshevskiy) [inline include/eve/module/core/regular/modf.hpp:25]: let's have a typedef for `eve::result_t<zip,T,T>`
`zipped`?

**Review by DenisYaroshevskiy — COMMENTED** [review]
Unfortuantely not just tuples but other math changes. I'm not qualified to review those, @jtlap  please have a look

File: include/eve/module/core/regular/impl/ifrexp.hpp (hunk 2 above)
Comment (jfalcou) [reply include/eve/module/core/regular/impl/ifrexp.hpp:37]: It is just a migration of the pedantic case in a single file. Nothing more.

File: include/eve/module/core/regular/modf.hpp (hunk 3 above)
Comment (jfalcou) [reply include/eve/module/core/regular/modf.hpp:25]: Maybe an alias `zipped<T..>` cause there is a bunch of variants.

**Comment by jfalcou** [comment]
> Unfortuantely not just tuples but other math changes. I'm not qualified to review those, @jtlap please have a look

The math part has been done under jtlap scrutiny and are mostly fix for issues that has native abi checked hide before.

File: include/eve/module/core/regular/modf.hpp (hunk 3 above)
Comment (DenisYaroshevskiy) [reply include/eve/module/core/regular/modf.hpp:25]: That's what meant, yes

**Review by DenisYaroshevskiy — APPROVED** [review]

**Pull request merged**